auto3dseg/notebooks/msd_datalist_generator.ipynb

```diff
@@ -0,0 +1,521 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Copyright (c) MONAI Consortium \n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\"); \n",
+    "you may not use this file except in compliance with the License. \n",
+    "You may obtain a copy of the License at \n",
+    "    http://www.apache.org/licenses/LICENSE-2.0 \n",
+    "Unless required by applicable law or agreed to in writing, software \n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS, \n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. \n",
+    "See the License for the specific language governing permissions and \n",
+    "limitations under the License. \n",
+    "\n",
+    "Created by Udbhav Ram - Cardenas Lab, University of Alabama at Birmingham for contribution towards the MONAI project \n"
```

Comment:
I am not sure if name/institution would conflict with the apache 2.0 license. Is it possible to remove it?

Reply:
ok, no problem, i'll make the change